services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java

```diff
@@ -279,9 +288,42 @@ public UnidirectionalFlow updateFlow(FlowDto flow) throws FlowNotFoundException
 
             flowRepository.createOrUpdate(currentFlow);
 
-            return Optional.of(forwardFlow);
+            return Optional.of(result.updatedFlow(forwardFlow).build());
```

Comment:
I think you need to add saving of `getPathComputationStrategy()` into `currentFlow`. like in lines 20 and 283

Reply:
Saving of `getPathComputationStrategy()` is added here: #3036